packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -286,6 +285,7 @@ export class QuoteRequestor {
             Object.keys(this._rfqtAssetOfferings).map(async url => {
                 if (this._makerSupportsPair(url, makerAssetData, takerAssetData)) {
                     const requestParams = {
+                        apiKey: options.apiKey,
```

Comment:
Curious - why did this have to get added?

Reply:
It didn't, thank you for catching this.  This was the result of my confusing the presence of an `apiKey` in the `TakerRequest` in `@0x/quote-server` as indicating that the server expects the API key as a query parameter, but I've since realized that that `TakerRequest` structure actually represents the _output_ of request parsing, at which point the header and query params have been merged into one structure.

Addressed in https://github.com/0xProject/0x-monorepo/pull/2582/commits/f7cd7110ea7dae6023127b9b6cb674a0cf1e1735